docusaurus/docs/dev-docs/configurations/database.md

```diff
@@ -535,3 +539,17 @@ $ GRANT ALL ON SCHEMA public TO my_strapi_db_user;
 ```
 
 :::
+
+### Alternative database drivers
+
+Strapi attempts to load database drivers for each client based on which packages have been added to the project.
+
+| client   | order of driver selected when available  |
```

Comment:
```suggestion
| client   | order of package selected when available  |
```
I think this might sound better, but I've been going back and forth on package vs driver I'm not sure anymore. Should it be "driver package"?